### DISCUSSION TITLE
Quick Question about Ghost Telemetry

### DISCUSSION THREAD

**Discussion by Louis-Land** [iNavFlight/inav#7400] [Q&A]
I do apologize about abusing the issues to ask this question. However I am trying to contract ImmersionRc about updating Ghost to fully support all the telemetry data like Crossfire and FrSky does. To make sure I get them the correct data. What fields are they current missing in order to support the Inav OpenTx Lua script and or what other important fields should they support?

Thank you,

Wyatt

**Comment by DzikuVx** [comment]
Quite a lot. There is no flight mode and attitude for example. Probably the best way is to compare what Crossfire offers with what Ghost offers.

**Reply by Louis-Land** [reply]
Hey Pawel, well I ended up buying an Happy Model ELRS since it supports the INav OpenTx Telemetry widget. I was able to compare the missing fields that way at least. I emailed ImmersionRC asking about adding those fields. The response from the email, they are planning to try to support more of INav telemetry in the next major release.

Thank you again for responses, I appreciate it.
